card-app/src/UserCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import getUsers from './getUsers';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: 1000,
+    maxWidth: 545,
+  },
+});
+
+const UserCard = () => {
+  const [user, setUser] = React.useState({
```

Comment:
This is kind of cool - when I did this, I used `null` as the default value for the user, but having the keys here lets you know what to expect to be included, which is nice. I'd probably use `null` for the property values instead of `loading...`, as the loading message will display kind of weird for things like the avatar. 

```js
{
  avatar: null,
  email: null,
  //...etc 
}